app/plugins/detect-overflows.js

```diff
@@ -0,0 +1,169 @@
+import { isFixed } from '../utilities/';
+
+export const commands = [
+  'detect overflows',
+  'overflow',
+]
+
+export const description = 'find elements that overflow the page body'
+
+const className = 'visbug-detect-overflows'
+
+export default function detectOverflows() {
+  document
+    .querySelectorAll('.' + className + ', visbug-hover')
+    .forEach((el) => el.remove())
+    document.querySelectorAll('*').forEach((el) => {
+    const isOverflowing = el.offsetWidth > document.documentElement.offsetWidth
+    const isFlag = el.classList.contains(className)
+    const alreadyHasFlag = el.lastChild && el.lastChild.className == className
+    if (isOverflowing && !isFlag && !alreadyHasFlag) {
+      createFlag(el)
```

Comment:
there's already flags and overlay elements to use! i appreciate this work here, but can you:
1. leverage the already existing elements instead please? See [the z-index plugin here](https://github.com/GoogleChromeLabs/ProjectVisBug/blob/main/app/plugins/zindex.js#L23), for an example of how to use visbug's elements.
2. move the "stay inside the viewport" logic to a separate PR so it can fix #159

I think making this plugin work more like the z-index one keeps things simpler. We may even be able to not use the label, but if we do, let's keep it short like `overflowing x`.

What do you think of trying to make this work with nested scrollers too? like, what if:
- if nothing is selected, perform the plugin on the body (like it does now)
- if an element is selected (which visbug provides when invoking the plugin), perform the plugin on the element. 

This let's a developer test the body and any nested scrollers they have, like scrolling in a `<main>` or scrollable sidenav menu. Thoughts?

Reply:
all sounds good, thanks!